src/constants.ts

```diff
@@ -37,6 +37,10 @@ export const IO_DEVNET_PROCESS_ID =
 export const ioDevnetProcessId = IO_DEVNET_PROCESS_ID;
 export const IO_TESTNET_PROCESS_ID =
   'agYcCFJtrMG6cqMuZfskIkFTGvUPddICmtQSBIoPdiA';
+
+// TODO: set ant registry ID with deployed contract
+export const ANT_REGISTRY_ID =
```

Comment:
don't support an env var here - let clients decide if they want to use something else